sparkbox/sparkle
# feat: Font System, Docs, and Tests

**Description**
Font Size, Font Weight, and Font Family have each of the following:
- Documentation
- Tests
- Function
- Mixin
- Utility

**To test:**
Run `npm test` to verify that tests pass
Run `npm run doc` followed by` python -m SimpleHTTPServer 3000` to spin up a server at http://localhost:3000 to check docs.

Known Issue: My documentation may need some fine tuning. Suggestions are welcomed.
